app/models/translatable.rb

```diff
@@ -0,0 +1,14 @@
+module Translatable
+  def en_name
+    self.name
+  end
+
+  def localized_name
```

Comment:
So the intention here was not quite to default to name for an unrecognised locale, but rather to ensure that a value is returned if none was found. I agree though, that behaviour probably is unexpected, so can be removed.